src/main/java/racing/Racing.java

```diff
@@ -25,4 +35,24 @@ private void updateCar() {
             participant.showDistance();
         }
     }
+
+    public void winner() {
+        int longest_distance = findLongestDistance();
```

Comment:
```suggestion
        int longestDistance = findLongestDistance();
```

다른 변수나 메소드명에 맞춰 낙타표기법을 사용해주세요.
강의 자료에 포함된 [컨벤션](https://google.github.io/styleguide/javaguide.html#s5-naming) 에도 낙타 표기법 사용을 권장하고 있습니다.